js/private/js_run_devserver.mjs

```diff
@@ -129,18 +136,15 @@ export function friendlyFileSize(bytes) {
     )
 }
 
-async function syncSymlink(src, dst, sandbox, exists) {
-    const srcWorkspacePath = src.slice(RUNFILES_ROOT.length + 1)
```

Comment:
Adding `file` (the runfiles-relative file path) avoids things like this `slice`, and we already had it on a few of the `sync*()` methods.

This specific line was invoked extremely often, other lines are rare/debug but make the code shorter.